__tests__/app.test.js

```diff
@@ -78,24 +78,58 @@ describe("Get", () => {
 				.get("/api/reviews/1")
 				.expect(200)
 				.then(({ body }) => {
-						expect(body['review']).toMatchObject({
-							review_id: 1,
-							title: 'Agricola',
-							review_body: 'Farmyard fun!',
-							designer: 'Uwe Rosenberg',
-							votes: 1,
-							category: 'euro game',
-							owner: 'mallionaire',
-							created_at: '2021-01-18T10:00:20.514Z',
+					expect(body["review"]).toMatchObject({
+						review_id: 1,
+						title: "Agricola",
+						review_body: "Farmyard fun!",
+						designer: "Uwe Rosenberg",
+						votes: 1,
+						category: "euro game",
+						owner: "mallionaire",
+						created_at: "2021-01-18T10:00:20.514Z",
+					});
+				});
+		});
+		test("200-responses with the correct item", () => {
+			return request(app)
+				.get("/api/reviews/1")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body["review"].review_id).toBe(1);
+				});
+		});
+	});
+	describe("api/reviews/review_id/comments", () => {
+		test("200-responses with an array of comment objects", () => {
+			return request(app)
+				.get("/api/reviews/2/comments")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body["comments"].length).toBe(3);
+					body["comments"].forEach((comment) => {
+						expect(comment).toMatchObject({
+							comment_id: expect.any(Number),
+							votes: expect.any(Number),
+							created_at: expect.any(String),
+							author: expect.any(String),
+							body: expect.any(String),
+							review_id: expect.any(Number),
 						});
+					});
+				});
```

Comment:
We also want to add the following tests for this end point:

Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999